### ISSUE TITLE
IPv6 Prefix <> IPv6 Display prefix ?

### ISSUE THREAD

**Issue by el-es** [SpriteLink/NIPAP#1289]
TL;DR question: Why are there different fields for actual and display prefix length, and 'display prefix' anyway?

NIPAP (web) will let me add a 128 long IPv6 host address, and show it as such on its edit window (click the cog->edit) (the prefix will be shown as /128 as intended)

But NIPAP (web) prefixes _list_ will list this as if it had a prefix length of 48 instead (it's a host assigned out of assignment of /48 length).

NIPAP cli will show why : 
nipap address list  - will show the same list as the nipap-web prefixes list (showing prefix length 48)

but 

nipap address view [address] will reveal there are in fact 2 fields :-

Prefix - which shows length 128 (as entered)
and
Display prefix - which shows length 48 (as in, part of the subnet it was assigned from)

Is there a reason why these are different in the 'list' and in the actual view?

nipap CLI client version: 0.29.9
pynipap version: 0.29.9
nipapd version: 0.29.9
(from debian [stretch,32bit]  packages)
Thanks in advance :)
(EDIT: installed from the github deb source not 'mainline', sorry for confusion here)

**Comment by garberg** [comment]
Yes, that's by design. In most cases when you add host prefixes under an assignment, what you probably want is the "display prefix" which shows what you're likely to actually configure on an interface as the network in use has prefix length /48 for example. That it's stored in NIPAP as a /128 is more of an implementation artifact.
